### PR TITLE
Updating covidcountyclient + a bit of import restructuring

### PR DESCRIPTION
**Pull request by ghop02**
I updated the client to the latest package and reran the updater - no data changed! 

One question i have, should we change the name of the updater/data all the way down the stack (maybe `s/cmdc/covidcountydata/g`? More than happy to, but also fine just getting this change in.

### CODE REVIEW

**Comment by ghop02** [comment]
[Upgrade to new Valorum client](https://trello.com/c/oioRVTlE/348-upgrade-to-new-valorum-client)

File: scripts/update_covid_county_data.py [hunk 1]
```diff
@@ -98,23 +94,23 @@ class Config:
...
-        cmdc_client = cmdc.Client(apikey=self.cmdc_key)
+        client = covidcountydata.Client(apikey=self.covid_county_data_key)
 
-        cmdc_client.covid_us()
-        df = cmdc_client.fetch()
+        client.covid_us()
+        df = client.fetch()
         # Transform FIPS from an int64 to a string of 2 or 5 chars. See
-        # https://github.com/valorumdata/cmdc.py/issues/3
+        # https://github.com/valorumdata/covid_county_data.py/issues/3
```
Comment (TomGoBravo) [inline scripts/update_covid_county_data.py:113]: https://github.com/valorumdata/cmdc.py/issues/3

File: scripts/update_covid_county_data.py [hunk 2]
```diff
@@ -186,11 +182,11 @@ def transform(self) -> pd.DataFrame:
...
     common_init.configure_logging()
     log = structlog.get_logger()
-    transformer = CmdcTransformer.make_with_data_root(
+    transformer = CovidCountyDataTransformer.make_with_data_root(
         DATA_ROOT, os.environ.get("CMDC_API_KEY", None), log,
     )
-    write_df_as_csv(
-        only_common_columns(transformer.transform(), log),
-        DATA_ROOT / "cases-cmdc" / "timeseries-common.csv",
+    common_df.write_df_as_csv(
```
Comment (TomGoBravo) [inline scripts/update_covid_county_data.py:188]: common_df.write_csv

**Comment by TomGoBravo** [comment]
This change does suggest that referring to this data as 'valorum' data may have given us a more stable name than 'cmdc'. :-)

**Pull request merged**